packages/aws-client/package.json

```diff
@@ -46,12 +46,12 @@
     "@cumulus/checksum": "6.0.0",
     "@cumulus/errors": "6.0.0",
     "@cumulus/logger": "6.0.0",
-    "aws-sdk": "^2.585.0",
+    "aws-sdk": "^2.814.0",
```

Comment:
There wasn't anything in the ticket, but isn't this functionally the same thing given the caret?   Is there something that's breaking < .814? 

I ask, because If we're *not* installing the newer minor version on build in CI, is something wrong with the image caching?